pyproject.toml

```diff
@@ -25,60 +26,61 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.7,<3.11"
 click = ">=7.1<=9.0"
-click-default-group = "*"
-colour = "*"
+click-default-group = "^1.2.2"
+colour = "^0.1.5"
 numpy = "^1.9"
-Pillow = "*"
-scipy = "*"
-tqdm = "*"
-pydub = "*"
+Pillow = "^8.4.0"
+scipy = "^1.7.3"
+tqdm = "^4.62.3"
+pydub = "^0.25.1"
 pygments = "*"
 rich = ">=6.0"
 pycairo = "^1.19"
 manimpango = "^0.3.0"
 networkx = "^2.5"
 decorator = "^5.0.7"
-setuptools = "*"
-importlib-metadata = {version = "*", python = "<3.8"}
-grpcio =  { version = "1.33.*", optional = true }
-grpcio-tools = { version = "1.33.*", optional = true }
-watchdog = "*"
+setuptools = "^60.1.0"
```

Comment:
Do we really need `setuptools` to be installed? Can you point out where it is used?

Reply:
I have no idea, I didn't add it; I only fixed the `*` notation.

Reply:
In that case, can we remove it?

Reply:
Does manim fail to run without this dependency?